README.md

```diff
@@ -6,120 +6,285 @@
+output: torch.Tensor = loss(prediction, target, data_range=1.)
+output.backward()
+```
+</p>
+</details>
 
-To compute measure or metric, for instance SSIM index, use lower case function from the library:  
+<!-- MS-SSIM EXAMPLES -->
+<details>
+<summary>Multi-Scale Structural Similarity (MS-SSIM)</summary>
+<p>
 
+To compute MS-SSIM index as measure, use lower case function from the library:
 ```python
 import torch
+from photosynthesis_metrics import multi_scale_ssim
 
-from photosynthesis_metrics import ssim
+prediction = torch.rand(3, 3, 256, 256)
+target = torch.rand(3, 3, 256, 256) 
+ms_ssim_index: torch.Tensor = multi_scale_ssim(prediction, target, data_range=1.)
+```
 
+In order to use MS-SSIM as a loss function, use corresponding PyTorch module:
+```python
+import torch
+from photosynthesis_metrics import MultiScaleSSIMLoss
 
-prediction = torch.rand(3, 3, 256, 256)
+loss = MultiScaleSSIMLoss()
+prediction = torch.rand(3, 3, 256, 256, requires_grad=True)
 target = torch.rand(3, 3, 256, 256)
-ssim_index = ssim(prediction, target, data_range=1.)
+output: torch.Tensor = loss(prediction, target, data_range=1.)
+output.backward()
 ```
+</p>
+</details>
 
-In order to use SSIM as a loss function, use corresponding PyTorch module:
+<!-- TV EXAMPLES -->
+<details>
+<summary>Total Variation (TV)</summary>
+<p>
 
+To compute TV as measure, use lower case function from the library:
 ```python
 import torch
+from photosynthesis_metrics import total_variation
 
-from photosynthesis_metrics import SSIMLoss
+data = torch.rand(3, 3, 256, 256) 
+tv: torch.Tensor = total_variation(data)
+```
 
+In order to use TV as a loss function, use corresponding PyTorch module:
+```python
+import torch
+from photosynthesis_metrics import TVLoss
 
-loss = SSIMLoss()
+loss = TVLoss()
 prediction = torch.rand(3, 3, 256, 256, requires_grad=True)
 target = torch.rand(3, 3, 256, 256)
-output = loss(prediction, target, data_range=1.)
+ouput: torch.Tensor = loss(prediction, target, data_range=1.)
 output.backward()
-``` 
+```
+</p>
+</details>
 
-<!-- ROADMAP -->
-## Roadmap
+<!-- MSID EXAMPLES -->
+<details>
+<summary>Multi-Scale Intrinsic Distance (MSID)</summary>
+<p>
 
-See the [open issues](https://github.com/photosynthesis-team/photosynthesis.metrics/issues) for a list of proposed 
-features (and known issues).
+Use `MSID` class to compute [MSID score](https://arxiv.org/abs/1905.11141) from image features, 
+pre-extracted from some feature extractor network: 
+```python
+import torch
+from photosynthesis_metrics import MSID
 
+msid_metric = MSID()
+prediction_feats = torch.rand(10000, 1024)
+target_feats = torch.rand(10000, 1024)
+msid: torch.Tensor = msid_metric(prediction_feats, target_feats)
+```
 
-<!-- CONTRIBUTING -->
-## Contributing
+If image features are not available, extract them using `_compute_feats` of `MSID` class. 
+Please note that `_compute_feats` consumes a data loader of predefined format. Refer to [docs](docs) for more details.
```

Comment:
This link doesn't work now, it will be fixed after merging into master?

Reply:
It is a placeholder for now. I assumed liking autogenerated documentation here, but it is not feasible since we don’t have one yet. Probably should remove it